animation_nodes/data_structures/meshes/mesh_data.pyx

```diff
@@ -132,11 +140,29 @@ cdef class Mesh:
         return list(self.vertexColorLayers.items())
 
     def getVertexColorLayerNames(self):
-        return list(self.vertexColorLayers.keys())    
-    
+        return list(self.vertexColorLayers.keys())
+
     def getVertexColors(self, str colorLayerName):
         return self.vertexColorLayers.get(colorLayerName, None)
-        
+
+    def getVertexLinkedEdges(self, long vertexIndex):
+        cdef IntegerList neighboursAmounts, neighboursStarts, neighbours, neighbourEdges
+        neighboursAmounts, neighboursStarts, neighbours, neighbourEdges = self.getVerticesLinkedEdges()
+
+        cdef int start, amount
+        amount = neighboursAmounts[vertexIndex]
+        start = neighboursStarts[vertexIndex]
```

Comment:
Don't forget `.data`. `neighboursAmounts.data[vertexIndex]`

Reply:
Okay. Should I also changed the name of the related functions in the `mesh_data.pyx` ?

Reply:
Yes, I think this would be a good idea.